src/widgets/ActivityDetails/ui/ActivityDetails.tsx

```diff
@@ -27,36 +31,21 @@ export const ActivityDetailsWidget = () => {
 
   const isActivityStarted = items.length > 0;
 
-  const {
-    activityDetails,
-    isLoading,
-    isError,
-    error,
-    appletDetails,
-    eventsRawData,
-    respondentMeta,
-  } = activityDetailsModel.hooks.useActivityDetailsQuery();
+  const { activityDetails, isLoading, isError, appletDetails, eventsRawData, respondentMeta } =
+    activityDetailsModel.hooks.useActivityDetailsQuery();
 
   if (isLoading) {
     return <AssessmentLoadingScreen />;
   }
 
-  if (isError) {
-    return (
-      <Box height="100vh" width="100%" display="flex" justifyContent="center" alignItems="center">
-        <span>
-          {context.isPublic ? t('additional.invalid_public_url') : error?.evaluatedMessage}
-        </span>
-      </Box>
-    );
-  }
+  if (!appletDetails || !activityDetails || !eventsRawData || isError) {
+    setTimeout(() => {
+      showErrorNotification(t('unabletoLoadActivity'));
```

Comment:
That's a good point. I'm not so sure that error message is completely valid. It says *"This assessment is no longer accepting respondents"*, but it would just be an invalid parameter. I think a generic error message is better than an incorrect one, but I'll wait to hear from a SS team member about it